Packs/URLHaus/Integrations/URLHaus/README.md

```diff
@@ -61,83 +70,28 @@ Retrieves URL information from URLhaus.
 | URLhaus.URL.Blacklist.Status | String | Status of the URL in the block list. | 
 | URLhaus.URL.Payload.VT.Link | String | Link to the VirusTotal report. | 
 
-
-#### Command Example
-```!url url="http://example.com/VMYB-ht_JAQo-gi/INV/99401FORPO/20673114777/US/Outstanding-Invoices/"```
-
+#### Command example
+```!url using-brand=URLhaus url=http://example.com/VMYB-ht_JAQo-gi/INV/99401FORPO/20673114777/US/Outstanding-Invoices/```
 #### Context Example
 ```json
 {
     "DBotScore": {
         "Indicator": "http://example.com/VMYB-ht_JAQo-gi/INV/99401FORPO/20673114777/US/Outstanding-Invoices/",
         "Reliability": "C - Fairly reliable",
-        "Score": 1,
+        "Score": 0,
         "Type": "url",
         "Vendor": "URLhaus"
     },
     "URL": {
         "Data": "http://example.com/VMYB-ht_JAQo-gi/INV/99401FORPO/20673114777/US/Outstanding-Invoices/"
-    },
-    "URLhaus": {
```

Comment:
now it's too little information :(
what I meant previously was that you should edit this manually and remove some of the output, so it won't be too long for the reader (but they still might want to see an informative example).